### ISSUE TITLE
 NVDA focus does not remain on the same column header "Short Description","Submission Status","Date Submitted".... when hit with "Enter" and the state of the button changes from ascending to descending and vice versa.

### ISSUE THREAD

**Issue by nirmalsahoo** [OfficeDev/office-ui-fabric-react#10434]
Environment Details:
Application Name: MSRC Web
Windows Version: Windows10
Versions : 
                Chrome Version 69.0.3497.100

Steps to Reproduce:
1.Start ScreenReader.
2.Launch "MSRC Web" Web Application or directly hit URL
3.Go to the HomePage i.e. Landing Page 
4.Go to the table-->data under the column headers "Date Submitted"-->Press Enter to sort.
5.Verify that after the button is hit,NVDA focus remains on the same column header button.
 

Actual:
NVDA focus does not remain on the same column header "Short Description","Submission Status","Date Submitted".... when hit with "Enter" and the state of the button changes from  ascending to descending and vice versa.
 
Soon after rhe header is hit with "Enter", the screen reader focus disappears and is not visible anymore , pressing "TAB" it goes to the cell of the grid and then it is visible for further navigation.
 
Expected:
NVDA focus should remain on the same column header "Short Description","Submission Status","Date Submitted".... when hit with "Enter" and the state of the button changes from  ascending to descending and vice versa.
 
Soon after the header is hit with "Enter", the screen reader focus should be present on the same column header and should be visible for further navigation.
 
User Impact:
User will not be able to track the follow of the focus.

**Comment by natalieethell** [comment]
After taking a closer look, it seems like the focus functionality remains even with the missing focus "highlight" from NVDA. For example, if I try to sort the column header again by pressing "Enter" or "Space", the column header onClick handler is triggered. Additionally, if I try to navigate by pressing 'Tab' or the arrow keys, navigation also still works as expected.

This seems more like an NVDA-specific issue than a Fabric issue, as the usability is all still there. Would you agree? Plus, Narrator + Edge provides the expected behavior.

Given the above, I would suggest opening an issue with NVDA. I'm not sure what else we can do on our end to keep the visual "highlight" from NVDA on a focused element.

Let us know if you have any further questions.
